data/data-pipeline/data_pipeline/etl/score/etl_score.py

```diff
@@ -384,7 +394,19 @@ def _prepare_initial_df(self) -> pd.DataFrame:
 
         census_tract_df = self._join_tract_dfs(census_tract_dfs)
 
-        # If GEOID10s are read as numbers instead of strings, the initial 0 is dropped,
+        # Drop tracts that don't exist in the 2010 tracts
+        pre_join_len = census_tract_df.shape[0] 
+
+        census_tract_df = census_tract_df.merge(
+            self.national_tract_df,
+            on="GEOID10_TRACT",
+            how="inner",
+        )
+
+        logger.info(
```

Comment:
This is me being CRAZY but I would assert the size is <= to make sure the join performed as expected -- it obviously should but 🤷

Reply:
that is a good call and i added it!